plugins/BEdita/Core/src/Utility/JsonSchema.php

```diff
@@ -86,6 +95,22 @@ public static function typeSchema($typeName)
         }
     }
 
+    /**
+     * Add revision information to schema
+     *
+     * @param array|bool $schema Schema array or `false`
+     * @return array|bool Schema with `revision` or `false`
+     */
+    protected static function addRevision($schema)
+    {
+        if (!is_array($schema)) {
+            return $schema;
+        }
+        $schema['revision'] = sprintf("%u", crc32(json_encode($schema)));
```

Comment:
Why CRC32? This is useful if we want to convert this value to a number to do some math with it, because it produces a 32-bit hash that can be safely converted to an integer on 64-bit machines, but if it has to be an opaque string we might use SHA1…

Reply:
Just to have a short, simple and fast hash. 
Might use other algorithms of course.